src/lib/components/Feedback/Loading.stories.svelte

```diff
@@ -0,0 +1,40 @@
+<script module lang="ts">
+	import { defineMeta } from '@storybook/addon-svelte-csf';
+	import type { ComponentProps } from 'svelte';
+	import Loading from './Loading.svelte';
+
+	const { Story } = defineMeta({
+		title: 'Feedback/Loading',
+		component: Loading,
+		argTypes: {
+			color: {
+				control: 'select',
+				options: [
+					'neutral',
+					'primary',
+					'secondary',
+					'accent',
+					'info',
+					'success',
+					'warning',
+					'error',
+				],
```

Comment:
I see some of them also used in the alert and other place, maybe it could be worth it to put them in an array or enums as UI tokens

Reply:
Same thing for the sizes tokens and anything you see used in common

Reply:
Just noticed my comment is on the stories file but I meant it to be on the component itself

Reply:
It did occur to me, there can be variation between components though, some will have ghost, others might ahve neutral, or not. So I just haven't made a decision for it yet.